client/src/components/Profile/Profile.jsx

```diff
@@ -61,7 +62,7 @@ const Profile = ({ user, history }) => (
 					<Button type="simple" clickHandler={() => history.push('/profile/edit')}>
```

Comment:
We should refactor `clickHandler`s to be a standalone functions outside of the JSX. That way we'll avoid unnecessary rerenders.